Palmer_McKay/app_base.htm

```diff
@@ -0,0 +1,95 @@
+<%inherit file="/homepage/templates/app_base.htm" />
+<%! from catalog import models as cmod %>
+
+<%block name="left">
+    <div id="category_list" class="col-sm-2 ">
+        <ul class="seo-list">
+            <a id="all" class="categories" href="/catalog/index"><li>All Products</li></a>
+            % for cat in cmod.Category.objects.order_by('name'):
+                <a class="categories" href="/catalog/index/${cat.id}"><li>${ cat.name }</li></a>
+            % endfor
+        </ul>
+    </div>
+</%block>
+
+<%block name="right">
```

Comment:
Right what? Nav bar? Top? Content? Footer?

Reply:
Yes, I should specify that is the right content